### ISSUE TITLE
Unable to run

### ISSUE THREAD

**Issue by ccpony** [DaveMBush/angular-book2#1]
Read your book, great job.  Enjoyed it very much. 

Cloned single branch step-25-cli-1.5.2

On running localhost:4200, generated this console error:

```
Uncaught Error: Can't resolve all parameters for ErrorHandler: (?).
    at syntaxError (webpack-internal:///../../../compiler/esm5/compiler.js:706)
    at CompileMetadataResolver._getDependenciesMetadata (webpack-internal:///../../../compiler/esm5/compiler.js:15913)
    at CompileMetadataResolver._getTypeMetadata (webpack-internal:///../../../compiler/esm5/compiler.js:15748)
    at CompileMetadataResolver._getInjectableMetadata (webpack-internal:///../../../compiler/esm5/compiler.js:15728)
    at CompileMetadataResolver.getProviderMetadata (webpack-internal:///../../../compiler/esm5/compiler.js:16088)
    at eval (webpack-internal:///../../../compiler/esm5/compiler.js:15999)
    at Array.forEach (<anonymous>)
    at CompileMetadataResolver._getProvidersMetadata (webpack-internal:///../../../compiler/esm5/compiler.js:15959)
    at CompileMetadataResolver.getNgModuleMetadata (webpack-internal:///../../../compiler/esm5/compiler.js:15527)
    at JitCompiler._loadModules (webpack-internal:///../../../compiler/esm5/compiler.js:34619)
```

Can you advise?  Thanks.

Jeff

**Comment by DaveMBush** [comment]
I'll have to take a look but it may be a couple of days.  It all worked when I wrote the book, so maybe something changed since then?

**Comment by DaveMBush** [comment]
I'm sorry, I switched to the branch, deleted node_module and reinstalled the packages (`npm install`).  The only problem I saw was that it now seems to want @angular-devkit/core installed, so I npm installed that.

`npm install save-dev @angular-devkit/core`

`npm start` ran find and I was able to browse the site without any trouble.

The only guesses I have is that you accidentally deleted something in one of your files and the pull didn't overwrite that file.  I've found that when I browse with the chrome developer tools open, it is more likely to take me to the typescript file where the problem is rather than showing me the resulting javascript like it does in your stack trace.
